ui/components/MesheryMeshInterface/helpers.js

```diff
@@ -255,10 +255,14 @@ export function formatString(text) {
 function jsonSchemaBuilder(schema, obj) {
   if (!schema) return
 
-  const uiDesc = "ui:description"
   // applying field template universally to every field type.
   obj["ui:FieldTemplate"] = CustomFieldTemplate;
```

Comment:
Every field in the rjsf follows our own CustomFieldTemplate, by which I mean we had defined how a field would look like in the ui. 
"additionalProperties" on the other side being a special case, where I am explicitly mention the description: " ", by which I suspect a bug on the rjsf side, I would create a scenario on codesandbox, so that I can create a issue related to this on rjsf repo . Or in case I would found a another way from the wizard hat.